keyboards/wilba_tech/wt65_h1/info.json

```
@@ -0,0 +1,12 @@
+{
+    "keyboard_name": "wilba.tech WT65-H1", 
+    "url": "https://wilba.tech", 
+    "maintainer": "Wilba", 
+    "width": 16, 
+    "height": 5, 
+    "layouts": {
+        "LAYOUT_all": {
```

This is one of our Community Layouts.

```suggestion
        "LAYOUT_65_ansi_blocker_tsangan": {
```